packages/react/src/components/AccountSettings/types.ts

```diff
@@ -33,3 +33,7 @@ export type AccountSettingsError<Props = {}> = React.ComponentType<
 
 /** Form specific types */
 export type FormValues = Record<string, string>;
+
+export type BlurredFields = Record<string, boolean>;
+
+export type ValidationError = Record<string, string[]>;
```

Comment:
nit: Wondering if it's possible/necessary to be more specific than `string[]`

Reply:
I do not think so, since validation errors are really dynamic (especially with custom validators)